cmd/nvidia_gpu/README.md

```diff
@@ -18,4 +18,9 @@ Examples of how driver installation needs to be done can be found at:
   - Installer code: https://github.com/GoogleCloudPlatform/container-engine-accelerators/blob/master/nvidia-driver-installer/ubuntu/entrypoint.sh
   - Installer daemonset: https://github.com/GoogleCloudPlatform/container-engine-accelerators/blob/master/nvidia-driver-installer/ubuntu/daemonset.yaml
 
+- For [Debian](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images#debian) (experimental):
```

Comment:
This link is 404.

Reply:
I cannot find a Google reference about Debian node image docs.